portfolio: setup and mobile first

gisachris opened the pull request:
in this PR i added:
new mobile first index page 
a styling sheet to the above index.html page
linter configurations

gisachris committed:
Added Linters Configuration Files

gisachris committed:
updated styling.css

gisachris committed:
updated styling.css

gisachris committed:
updated styling.css file

gisachris committed:
updated css file to match linters

gisachris committed:
update about stylelint to styling css file

gisachris committed:
update to stylesheet

gisachris committed:
Update styling.css

gisachris committed:
Update styling.css

gisachris committed:
Update styling.css

gisachris committed:
update README file

gisachris committed:
Update README.md

gisachris committed:
Update README.md

gisachris committed:
Update README.md

gisachris committed:
Create MIT.md

gisachris committed:
Update README.md

osamaashraf6 commented on index.html:
```diff
@@ -0,0 +1,66 @@
+  <body>
+    <div class="container">
+      <section class="section1">
+        <header>
```
- [ ] Kindly it will be better 😃 if you remove the parent section of the header and just put the header tag please try to put the header tag directly without section this is for semantics reasons you can read [Here](https://www.w3schools.com/html/html5_semantic_elements.asp) to know about the semantics elements and how to use them 👌💯

osamaashraf6 commented on index.html:
```diff
@@ -0,0 +1,66 @@
+      <section class="section1">
+        <header>
+          <nav class="navigation">
+            <section class="logo">
+              <a href="index.html" class="logo-text touch rough">my logo</a>
+            </section>
```
- [ ] Kindly 😃 try not to put the logo inside the section tag because the section tag is not used here, so please try to read about the usage of semantics [here](https://www.w3schools.com/html/html5_semantic_elements.asp) to get more information 💯

| The section tag| 
| --- |
|    ![image](https://user-images.githubusercontent.com/83360636/227517776-33e04672-c2d7-4fc5-917b-ad2aed695620.png)  |

gisachris committed:
Added Index Page And Style Sheet

osamaashraf6 commented on index.html:
```diff
@@ -0,0 +1,66 @@
+            <section class="logo">
+              <a href="index.html" class="logo-text touch rough">my logo</a>
+            </section>
+            <section class="item-nav">
+              <a href="menu-items" class="menu-bar touch rough"><img src="mobile tag.png" alt="menu bar"></a>
+              <p class="menu-items">
+                <section class="items touch rough">
+                  <p class="item1">portfolio</p>
+                  <p class="item2">about</p>
+                  <p class="item3">contact</p>
+                </section>
+              </p>
+            </section>
+
+            <section class="mail-button items">
+              <p class="mail touch rough"><img src="Union.png" alt="mail link"></p>
+            </section>
+          </nav>
+        </header>
+      </section>
+      <section class="section2">
+        <article class="intro-text">
+          <p class="text1">Hey There.<br>I'm Chris.</p>
+          <p class="text2">I'm a Software Developer</p>
+          <p class="text3">I can help you build a product , feature <br> or
+            website. Look through some of my <br> work and experience! If you
+            like what you <br> see and have a project you need coded,<br>
+            don’t hesitate to contact me.</p>
+        </article>
+        <article class="icons touch rough">
+          <p class="medium"><img src="medium.svg" alt="medium site"
+              class="icon-sizing"></p>
+          <p class="github"><img src="github.svg" alt="github site"
+              class="icon-sizing"></p>
+          <p class="angellist"><img src="angel list.svg" alt="angellist site"
+              class="icon-sizing"></p>
+          <p class="linkedn"><img src="linkedn.png" alt="linkedn site"
+              class="icon-sizing"></p>
+          <p class="twitter"><img src="twitter.png" alt="twitter site"
```
- [ ] I see that you used the semantic elements wrongly, so I instead of writing replace this with that I make to to you like a diagram to understand it and convert it into a code to better understand the structure of the page 😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Diagram design| Another diagram|
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/227523970-5134d3e8-5672-46ba-b1f3-18686fab1405.png)   | ![image](https://user-images.githubusercontent.com/83360636/227525106-52271a7b-8c91-4851-968a-dc9fea19345b.png)|

osamaashraf6 commented on index.html:
```diff
@@ -0,0 +1,66 @@
+            <section class="logo">
+              <a href="index.html" class="logo-text touch rough">my logo</a>
+            </section>
+            <section class="item-nav">
+              <a href="menu-items" class="menu-bar touch rough"><img src="mobile tag.png" alt="menu bar"></a>
+              <p class="menu-items">
+                <section class="items touch rough">
+                  <p class="item1">portfolio</p>
+                  <p class="item2">about</p>
+                  <p class="item3">contact</p>
+                </section>
+              </p>
+            </section>
+
+            <section class="mail-button items">
+              <p class="mail touch rough"><img src="Union.png" alt="mail link"></p>
+            </section>
+          </nav>
+        </header>
+      </section>
+      <section class="section2">
+        <article class="intro-text">
+          <p class="text1">Hey There.<br>I'm Chris.</p>
+          <p class="text2">I'm a Software Developer</p>
+          <p class="text3">I can help you build a product , feature <br> or
+            website. Look through some of my <br> work and experience! If you
+            like what you <br> see and have a project you need coded,<br>
+            don’t hesitate to contact me.</p>
+        </article>
+        <article class="icons touch rough">
+          <p class="medium"><img src="medium.svg" alt="medium site"
+              class="icon-sizing"></p>
+          <p class="github"><img src="github.svg" alt="github site"
+              class="icon-sizing"></p>
+          <p class="angellist"><img src="angel list.svg" alt="angellist site"
+              class="icon-sizing"></p>
+          <p class="linkedn"><img src="linkedn.png" alt="linkedn site"
```
- [ ] I see 😃 that the font size of the text on the page is like :
- The logo
- The h1
- The h2
- The paragraph 
their fonts are not like the Figma, so please try to stick the Figma to make the portfolio nice like the Figma design 😊
- [ ] Also consider increasing the margin around the section to make the content like the Figma design 😃


| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/227526067-f03fbdc6-f406-4f61-abb9-ed61c446cc5d.png)    |   ![image](https://user-images.githubusercontent.com/83360636/227526245-7b27be8f-131e-4a2d-8965-d5ecaaf611be.png)    |

gisachris replied:
since my last PR i have adjusted all the above by :
properly using semantics
adding desktop version

osamaashraf6 reviewed (changes requested):
Hi @gisachris 👋, 

Good job so far! 👏🥳
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

![you-are-near](https://media.giphy.com/media/SSR17K6yZWvGCYrfUh/giphy.gif)

## Highlights ✔️


- [x] No linter errors. 💯
- [x] Good usage in correcting Git flow
- [x] Nice design.




## Required Changes ♻️
- Check the comments under the review😊⤵️

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

gisachris committed:
Update styling.css

gisachris committed:
Update styling.css

gisachris committed:
Update styling.css

gisachris committed:
Update .gitignore

gisachris committed:
Create style.css

gisachris committed:
Update .gitignore

gisachris committed:
Update styling.css

gisachris committed:
Update index.html

gisachris committed:
Update styling.css

divinecharlotte commented on style.css:
```diff
@@ -0,0 +1,230 @@
+  list-style-type: none;
+}
+
+.icons {
+  display: flex;
+  flex-direction: column;
+  order: 10;
+  transform: rotate(0deg);
+  gap: 1vh;
+  position: absolute;
+  top: 35vh;
+  left: 0vh;
+  right: 0vh;
+  padding-left: 0vh;
+  margin-left: 0vh;
+}
+
+.medium {
+  color: #cfcfcf;
+  order: 0;
+  display: inline-flex;
+  width: auto;
+  height: auto;
+}
+
+.github {
+  color: #cfcfcf;
+  order: 1;
+}
+
+.angellist {
+  color: #cfcfcf;
+  order: 2;
+}
+
+@media screen and (min-width: 768px) {
+  .my-image {
+    content: url("image-large.jpg");
+  }
+}
```
- [ ]as requested by the previous reviewer kindly consider sticking to the provided figma design as much as possible make sure The section uses the font and colors (background and font) defined for the template to make a good final design kindly check on the images below
- your design 👇 
![Capture](https://user-images.githubusercontent.com/60146030/227700419-143ba1d9-8b03-49cf-9b49-0ccc7cc20f72.PNG)
- provided design 👇 
![c2](https://user-images.githubusercontent.com/60146030/227700439-2744a2dc-2b6f-430a-8d40-bfc5cdaa47e6.PNG)

divinecharlotte commented on index.html:
```diff
@@ -0,0 +1,63 @@
+          <li class="twitter"><img src="twitter.png" alt="twitter site"
+              class="icon-sizing"></li>
+          </ul>
+        <article class="art">
+          <p class="flow"><img src="illustration-header-mobile.svg" alt="some
+              art" class="art-i"></p>
+        </article>
```
- [ ] kindly consider adding this image as a background image as its a decoration this will increase your CSS skills on how to use an image as background image

divinecharlotte commented on index.html:
```diff
@@ -0,0 +1,63 @@
+          </nav>
+        </header>
+      </section>
+      <main class="section2">
+        <article class="intro-text">
+          <p class="text1">Hey There.<br>I'm Chris.</p>
+          <p class="text2">I'm a Software Developer</p>
+          <p class="text3">I can help you build a product , feature <br> or
+            website. Look through some of my <br> work and experience! If you
+            like what you <br> see and have a project you need coded,<br>
+            don’t hesitate to contact me.</p>
+        </article>
+        <ul class="icons touch rough">
+          <li class="medium"><img src="medium.svg" alt="medium site"
+              class="icon-sizing"></li>
+          <li class="github"><img src="github.svg" alt="github site"
+              class="icon-sizing"></li>
+          <li class="angellist"><img src="angel list.svg" alt="angellist site"
+              class="icon-sizing"></li>
+          <li class="linkedn"><img src="linkedn.png" alt="linkedn site"
+              class="icon-sizing"></li>
+          <li class="twitter"><img src="twitter.png" alt="twitter site"
+              class="icon-sizing"></li>
+          </ul>
+        <article class="art">
+          <p class="flow"><img src="illustration-header-mobile.svg" alt="some
+              art" class="art-i"></p>
+        </article>
+      </main>
```
- [ ] kindly consider making sure you add all The content in a section tag and its aligned with flexbox.
* This gives us a lot of options to align all our elements, plus, it's responsiveness

divinecharlotte commented on index.html:
```diff
@@ -0,0 +1,63 @@
+      </section>
+      <main class="section2">
+        <article class="intro-text">
+          <p class="text1">Hey There.<br>I'm Chris.</p>
```
- [ ] kindly consider adding this in h1 tag as its the main title of the page and it will be more clear to the readers

divinecharlotte reviewed (changes requested):
Hi @gisachris  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- clean code ✔️ 
- Descriptive PR ✔️ 

## Required Changes ♻️
_Check the comments under the review._

![](https://media.giphy.com/media/tKyrc0NTlEcGWdThp4/giphy.gif)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

gisachris committed:
added the required fonts

gisachris committed:
added last font

gisachris committed:
Update .stylelintrc.json

gisachris committed:
Update .stylelintrc.json

gisachris committed:
Update index.html

gisachris committed:
Update styling.css

gisachris committed:
Update styling.css

gisachris committed:
Update styling.css

gisachris commented:
i have changed the following in this project:
change page semantics to allow it to be more responsive
changing the image on page to be set as a background as requested
added proper fonts 
correctly placed items on the page

tsohleDev commented on index.html:
```diff
@@ -0,0 +1,47 @@
+  </head>
+  <body>
+    <div class="container">
+          <nav class="navigation">
```
- [ ] I think it is better if you only wrap the hamburger in the ``nav`` tag because the ``nav`` tag is used for wrapping elements that navigate and the logo does not navigate the app 😉

tsohleDev commented on index.html:
```diff
@@ -0,0 +1,47 @@
+    <link href="Roboto-Regular.ttf" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container">
+          <nav class="navigation">
+              <a href="index.html" class="logo-text touch rough">my logo</a>
+              <a href="menu-items" class="menu-bar touch rough"><img src="mobile tag.png" alt="menu bar"></a>
+          </nav>
```
- [ ] I suggest that you use the ``header`` tag rather than a ``div``, because its a good practice to structure your HTML accordingly this also helps it become more responsive 👍

tsohleDev commented on index.html:
```diff
@@ -0,0 +1,47 @@
+              <a href="index.html" class="logo-text touch rough">my logo</a>
+              <a href="menu-items" class="menu-bar touch rough"><img src="mobile tag.png" alt="menu bar"></a>
+          </nav>
+      <main class="section2">
+        <article class="intro-text">
```
- [ ] I kindly suggest that you use the ``section`` tag inside the main to wrap up all the content in the ``main``, because it is very beneficial to organize your HTML with proper tags 👍

tsohleDev commented on style.css:
- [ ] Kindly look at the video to see an example of how you would make your web more responsive, thus easier to style

https://user-images.githubusercontent.com/102636489/227728786-cf7d3bc1-9abf-421c-ba59-9f825fb18dac.mp4

tsohleDev reviewed (changes requested):
## Hi Micronaut @gisachris 

Good job so far!
There are some issues that you still need to work on.

![163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856](https://user-images.githubusercontent.com/97100091/191558701-8ed50201-3123-435c-89d8-29edcb66c64d.gif)

## Highlights!
- README template ✔️ 
- Descriptive commits ✔️ 
- Good branch name  ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- All requirements met ❌

Hello ✋, you have done well creating your project 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Compliment
_Every comment with the [COMPLIMENT] prefix is not  a request, just a compliment at your skills_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@tsohleDev` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

gisachris committed:
Update index.html

gisachris committed:
Update styling.css

gisachris commented:
i have done everything the last code reviewer has asked to be changed including: 
adding proper semantics to the page

gisachris committed:
Update styling.css

christianbiring1 reviewed (approved):
Hi @gisachris , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS:APPROVED :tada: 

## Highlights
- [ ] All linter checks are green :heavy_check_mark: 
- [ ] All suggested changes were implemented :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
